google-cloud-language/lib/google-cloud-language.rb

```diff
@@ -110,7 +110,7 @@ def self.language project = nil, keyfile = nil, scope: nil, retries: nil,
       end
       Google::Cloud::Language::Project.new(
         Google::Cloud::Language::Service.new(
-          project, credentials, retries: retries, timeout: timeout))
+          project, credentials, timeout: timeout, client_config: client_config))
```

Comment:
Instead of doing a straight pass-thru, it might be nice to pass in a file object, IO object, or file path (string) and have it parse the JSON and pass it in. If GAX doesn't want to add those capabilities we could add them here.